feature/src/common/frame.c

```diff
@@ -219,3 +510,64 @@ int get_frame_offset(const char *fmt, int w, int h, size_t *offset)
     return 0;
 }
 
+int get_color_resolution(enum VmafPixelFormat fmt, unsigned int w, unsigned int h, unsigned int *w_u, unsigned int *h_u, unsigned int *w_v, unsigned int *h_v) {
```

Comment:
you've been calling them "color" (among other functions). I think it is a bit sloppy. Can we use "chroma" instead?

Reply:
Fixed all "color" to "chroma"